src/library_sdl.js

```diff
@@ -2431,7 +2437,15 @@ var LibrarySDL = {
       }
 
       var totalSamples = SDL.audio.samples*SDL.audio.channels;
-      SDL.audio.bytesPerSample = (SDL.audio.format == 0x0008 /*AUDIO_U8*/ || SDL.audio.format == 0x8008 /*AUDIO_S8*/) ? 1 : 2;
```

Comment:
Note that this is the only place where where the format 0x8008 (AUDIO_S8) was mentioned, however, if AUDIO_S8 would have been used in SDL_OpenAudio it would have thrown an "Invalid SDL audio format" exception, since only AUDIO_U8 (0x0008) and AUDIO_S16LSB (0x8010) is handled there.

For this reason I have removed AUDIO_S8 in the if-else block I wrote as replacement for the trigraph.